python/TestHarness/testers/Tester.py

```diff
@@ -71,7 +72,7 @@ def validParams():
         params.addParam('party',         ['ALL'], "A test that runs only if Party (partitioner) is available via PETSc ('ALL', 'TRUE', 'FALSE')")
         params.addParam('ptscotch',      ['ALL'], "A test that runs only if PTScotch (partitioner) is available via PETSc ('ALL', 'TRUE', 'FALSE')")
         params.addParam('slepc',         ['ALL'], "A test that runs only if SLEPc is available ('ALL', 'TRUE', 'FALSE')")
-        params.addParam('unique_id',     ['ALL'], "A test that runs only if libmesh is configured with --enable-unique-id ('ALL', 'TRUE', 'FALSE')")
+        params.addParam('unique_id',     ['ALL'], "Deprecated. Use --unique_ids instead.")
```

Comment:
let's do it the other way around. In MOOSE/Bison `unique_id` is used 104 times, and `unique_ids` only twice!